ucsm_apis/admin/callhome.py

```diff
@@ -0,0 +1,397 @@
+                                    caller="call_home_profile_email_exist")
+    except UcsOperationError:
+        return (False, None)
+    mo_exists = mo.check_prop_match(**kwargs)
+    return (mo_exists, mo if mo_exists else None)
+
+
+def call_home_profile_email_remove(handle, profile_name, email):
+    mo = call_home_profile_email_get(handle, profile_name, email,
+                                caller="call_home_profile_email_remove")
+    handle.remove_mo(mo)
+    handle.commit()
+
+
+def call_home_policy_create(handle, cause, admin_state="enabled",
+                            name=None, descr=None, **kwargs):
+    mo = CallhomePolicy(parent_mo_or_dn=_base_dn,
+                        cause=cause,
+                        admin_state=admin_state,
+                        name=name,
+                        descr=descr)
+
+    mo.set_prop_multiple(**kwargs)
+    handle.add_mo(mo, True)
+    handle.commit()
+    return mo
+
+
+def call_home_policy_get(handle, cause, caller="call_home_policy_get"):
+    dn = _base_dn + "/policy-" + cause
+    mo = handle.query_dn(dn)
+    if mo is None:
+        raise UcsOperationError(caller,
+                            "Callhome Policy '%s' does not exist" % dn)
+    return mo
+
+
+def call_home_policy_exist(handle, cause, **kwargs):
+    try:
+        mo = call_home_policy_get(handle, cause,
+                                  caller="call_home_policy_exist")
+    except UcsOperationError:
+        return (False, None)
+    mo_exists = mo.check_prop_match(**kwargs)
+    return (mo_exists, mo if mo_exists else None)
+
+
+def call_home_policy_modify(handle, cause, **kwargs):
+    mo = call_home_policy_get(handle, cause, caller="call_home_policy_modify")
+    mo.set_prop_multiple(**kwargs)
+    handle.set_mo(mo)
+    handle.commit()
+    return mo
+
+
+def call_home_policy_remove(handle, cause):
+    mo = call_home_policy_get(handle, cause, caller="call_home_policy_remove")
+    handle.remove_mo(mo)
+    handle.commit()
+
+
+def call_home_system_inventory_configure(handle,
+                                         admin_state="on",
+                                         interval_days="30",
+                                         time_of_day_hour="0",
+                                         time_of_day_minute="0",
+                                         maximum_retry_count="1",
+                                         poll_interval_seconds="300",
+                                         retry_delay_minutes="10",
+                                         minimum_send_now_interval_seconds="5",
+                                         send_now="no",
+                                         **kwargs):
+    dn = _base_dn + "periodicsysteminventory"
+    mo = handle.query_dn(dn)
+    if mo is None:
+        raise UcsOperationError("call_home_system_inventory_configure",
+                        "Callhome system inventory '%s' does not exist." % dn)
+
+    mo.admin_state = admin_state
```

Comment:
can you please try using the dictionary format and do a `set_prop_multiple(**args)`